index.html

```diff
@@ -5164,6 +5164,116 @@ <h2 id="AudioWorklet-concepts">
           </dd>
         </dl>
         <section>
+          <h2 id="AudioWorkletGlobalScope">
+            The <dfn>AudioWorkletGlobalScope</dfn> interface
+          </h2>
+          <p>
+            This special execution context is designed to enable the
+            generation, processing, and analysis of audio data directly using
+            JavaScript in the audio <a>rendering thread</a>. The user-supplied
+            script code is evaluated in this scope to define an
+            <a>AudioWorkletProcessor</a>.
+          </p>
+          <p>
+            <a>AudioWorkletGlobalScope</a> has a <dfn>node name to processor
+            definition map</dfn>. Initially it is empty and populated when
+            <code>registerProcessor</code> method is called.
+          </p>
+          <dl title="callback VoidFunction = void" class="idl"></dl>
+          <dl title=
+          "[Global=(Worklet, AudioWorklet), Exposed=AudioWorklet] interface AudioWorkletGlobalScope : WorkletGlobalScope"
+          class="idl">
+            <dt>
+              void registerProcessor ()
+            </dt>
+            <dd>
+              <p>
+                Registers a class definition derived from
+                <a>AudioWorkletProcessor</a>.
+              </p>
+              <p>
+                When the <a><code>registerProcessor(<em>name</em>,
+                <em>processorCtor</em>)</code></a> method is called, the user
+                agent must run the following steps:
+              </p>
+              <ol>
+                <li>If the <code><em>name</em></code> exists as a key in the
+                <a>node name to processor definition map</a>, throw a
+                <code>NotSupportedError</code> and abort all these steps.
```

Comment:
The reason behind this, this method can get called anytime. So if we allow the dynamic redefinition of the processor, this becomes the data race. For example, the processor A('foo') is rendering the audio and its definition gets overwritten by the processor B('foo').

I'd say, we block the redefinition for V1. This is what Worklet spec does at the moment. This explanation can be a part of the spec as an informative section, if you will.